packages/xo-web/src/common/xo/index.js

```diff
@@ -1322,12 +1322,14 @@ export const createVms = (args, nameLabels, cloudConfigs) =>
 export const getCloudInitConfig = template =>
   _call('vm.getCloudInitConfig', { template })
 
+export const pureDeleteVm = vm => _call('vm.delete', { id: resolveId(vm) })
```

Comment:
It can takes props.

Reply:
`vm.delete` can takes other props https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-server/src/api/vm.js#L390

Reply:
Do you mean other arguments ^^, `props` is a keyword belongs only to `React` :p